src/dir.targets

```diff
@@ -94,8 +94,18 @@
   <Import Project="$(ToolsDir)publishtest.targets" Condition="Exists('$(ToolsDir)publishtest.targets')" />
 
   <PropertyGroup>
+    <IsInnerLoopTestProject Condition="'$(IsInnerLoopTestProject)'==''">True</IsInnerLoopTestProject>
+    <IsOuterLoopTestProject Condition="'$(IsOuterLoopTestProject)'==''">True</IsOuterLoopTestProject>
+
     <IsTestProject Condition="'$(IsTestProject)'=='' And $([System.String]::new('$(MSBuildProjectName)').EndsWith('.Tests'))">True</IsTestProject>
-    <RunTestsForProject Condition="'$(RunTestsForProject)'=='' and '$(IsTestProject)'=='True'">True</RunTestsForProject>
+    <IsTestProject Condition="'$(IsInnerLoopTestProject)'!='True' And '$(IsOuterLoopTestProject)'!='True'">False</IsTestProject>
```

Comment:
This now requires that ever test project set one or the other? What if they are both and don't care? I don't want to force people to have to set this.